yeastphenome/apps/conditions/views.py

```diff
@@ -78,7 +78,26 @@ def browse(request):
     """Browse dataest by condition names (and size by count)"""
     qs = (
         ConditionType.objects.all()
-        .annotate(number_of_datasets=Count("condition__conditionset__dataset"))
+        .annotate(
```

Comment:
Yeah, for some reason `Count` conditional on `exclude` didn't work for me either. But `Count` conditional on `filter` did. So maybe this would be a shorter / slightly more elegant solution?

`qs = ConditionType.objects.annotate(number_of_datasets=Count('condition__conditionset__dataset', filter=~Q(condition__conditionset__dataset__paper__latest_data_status__status__name='not relevant'))).order_by('-number_of_datasets')`

Reply:
Just tested locally, that seems to do the trick!

```python
qs2 = ConditionType.objects.annotate(number_of_datasets=Count('condition__conditionset__dataset', fil
    ...: ter=~Q(condition__conditionset__dataset__paper__latest_data_status__status__name='not relevant'))).or
    ...: der_by('-number_of_datasets')                                                                        

# If we hadn't filtered, we would see 6846
qs2.first().number_of_datasets                                                                       
6821
```
I'll update the PR.